gapis/resolve/resource_data.go

```diff
@@ -180,6 +182,19 @@ func (r *ResourceDataResolvable) Resolve(ctx context.Context) (interface{}, erro
 // Pipelines resolves the data of the currently bound pipelines at the specified
 // point in the capture.
 func Pipelines(ctx context.Context, p *path.Pipelines, r *path.ResolveConfig) (interface{}, error) {
+	cmd, err := Cmd(ctx, p.After, r)
+	if err != nil {
+		return nil, err
+	}
+	state, err := GlobalState(ctx, &path.GlobalState{After: p.After}, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if !cmd.CmdFlags(ctx, api.CmdID(p.After.Indices[0]), state).IsExecutedDraw() || len(p.After.Indices) == 1 {
```

Comment:
Let me know if there is more proper way to make sure only an executing (not recording) draw is being selected. So far, my tests are fine with this change.

Reply:
Nope, this is exactly what I would have done. We should probably add a helper method somewhere eventually.